.evergreen/run-big-endian-test.sh

```diff
@@ -1,5 +1,22 @@
-#!/usr/bin/env bash
+#! /usr/bin/env bash
```

Comment:
I know this is completely insignificant but that makes it even more interesting how this change became part of the diff :) Did some autoformatter do this?